api/Notification.json

```
@@ -15,7 +15,7 @@
             }
           ],
           "chrome_android": {
-            "version_added": false
+            "version_added": "44"
```

Thanks to your PR, I discovered that the whole `api.Notification` data for Chrome for Android needs some work. For now, I'd ask you to you to change this to `true` (see https://github.com/mdn/browser-compat-data/issues/7923#issuecomment-778359481 if you want all the detail, but it's not directly relevant to this PR).

```suggestion
            "version_added": true
```